src/highdicom/sr/content.py

```diff
@@ -707,6 +707,30 @@ def from_dataset(
         return cast(SourceSeriesForSegmentation, item)
 
 
+class CoordinatesForMeasurement(ScoordContentItem):
+
+    """Content item representing spatial coordinates of a measurement"""
+
+    def __init__(
+            self,
+            graphic_type: Union[GraphicTypeValues, str],
+            graphic_data: np.ndarray,
+            source_image: SourceImageForRegion,
```

Comment:
This has nothing to do with ImageRegion, but I reused `SourceImageForRegion` here as it needs to be an image with `SELECTED FROM` relationship. Not sure if there's a better option.